src/add.spec.js

```diff
@@ -0,0 +1,26 @@
+import add from "./add"
+
+// The add function should be tested to ensure that the addition of the provided integers & 
+// floating-point numbers provides the mathematically correct result. 
+
+describe('Test suite for "add"-function', () => {
+    it('Should return 5 on input 3 and 2', () => {
+        expect(add(3, 2)).toBe(5)
+    })
+    it('Should return 5 on input 3 and 2', () => {
+        expect(add(3, '2')).not.toBe(5)
```

Comment:
I like the test in general, but note that the jsdoc comment on the add function asks for the number type not string. So if it fails this could be considered still correct - even through I would expect an error to be thrown. For me we can keep it with the argumentation that a user will likely pass it string or a user input as string without being aware of the type expectation.